src/sentry/migrations/0026_exporteddata.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2020-01-17 23:14
```

Comment:
Here's the SQL for this script:

```
BEGIN;
--
-- Create model ExportedData
--
CREATE TABLE "sentry_exporteddata" ("id" bigserial NOT NULL PRIMARY KEY, "data_id" varchar(32) NOT NULL UNIQUE, "created_at" timestamp with time zone NOT NULL, "finished_at" timestamp with time zone NULL, "expired_at" timestamp with time zone NULL, "storage_url" varchar(200) NULL, "query_type" varchar(32) NOT NULL, "query_info" text NOT NULL, "organization_id" bigint NOT NULL, "user_id" integer NOT NULL);
ALTER TABLE "sentry_exporteddata" ADD CONSTRAINT "sentry_expor_organization_id_0be1ce88_fk_sentry_organization_id" FOREIGN KEY ("organization_id") REFERENCES "sentry_organization" ("id") DEFERRABLE INITIALLY DEFERRED;
ALTER TABLE "sentry_exporteddata" ADD CONSTRAINT "sentry_exporteddata_user_id_948c0003_fk_auth_user_id" FOREIGN KEY ("user_id") REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED;
CREATE INDEX "sentry_exporteddata_26b2345e" ON "sentry_exporteddata" ("organization_id");
CREATE INDEX "sentry_exporteddata_e8701ad4" ON "sentry_exporteddata" ("user_id");
CREATE INDEX "sentry_exporteddata_data_id_b3b1ae98_like" ON "sentry_exporteddata" ("data_id" varchar_pattern_ops);
COMMIT;
```

Reply:
SQL looks good, afaik it's safe to add foreign keys to both user and organization